jobs/snort-config/spec

```diff
@@ -5,6 +5,12 @@ provides:
   type: config
   properties:
   - snort.rules
+
+consumes:
+- name: snort_connection
```

Comment:
@JasonTheMain this is probably not needed?

Reply:
testing

Reply:
Looks good now